src/linkify-matrix.ts

```diff
@@ -196,15 +203,15 @@ matrixLinkify.MATRIXTO_MD_LINK_PATTERN =
 matrixLinkify.MATRIXTO_BASE_URL= baseUrl;
 
 matrixLinkify.options = {
-    events: function(href, type) {
+    events: function(href: string, type: Type | string): Partial<GlobalEventHandlers> {
         switch (type) {
-            case "url": {
+            case Type.URL: {
                 // intercept local permalinks to users and show them like userids (in userinfo of current room)
                 try {
                     const permalink = parsePermalink(href);
                     if (permalink && permalink.userId) {
                         return {
-                            click: function(e) {
+                            onclick: function(e) {
```

Comment:
not sure these are meant to be remapped to `onclick` given the library... would be good to see documentation on this.